Makefile

```diff
@@ -18,24 +18,40 @@
 # NOTE: git-version-gen will generate a value for VERSION, unless you override it.
 IMAGE_TAG_BASE ?= ghcr.io/nearnodeflash/nnf-mfu
 
+# CONTAINER_TOOL defines the container tool to be used for building images.
+# Be aware that the target commands are only tested with Docker which is
+# scaffolded by default. However, you might want to replace it to use other
+# tools. (i.e. podman)
+CONTAINER_TOOL ?= docker
+
+# Which tag to pull from the cray lustre source code.
+# **Note:** This must also be set in .github/workflows/main.yaml for both
```

Comment:
Set it the same way I set OPENMPI_VERSION so you don't have to set it in the workflow.

Reply:
Thanks.